modules/space/rendering/renderableorbitalkepler.cpp

```diff
@@ -548,20 +548,22 @@ void RenderableOrbitalKepler::render(const RenderData& data, RendererTasks&) {
         glLineWidth(_appearance.trailWidth);
 
         glBindVertexArray(_vertexArray);
+
```

Comment:
```suggestion
```

Would say to not have the empty lines to group the two `glBindVertexArray` and the `glMultiDrawArrays` call together